### DISCUSSION TITLE
Only printing the first page of a multiple page pdf.

### DISCUSSION THREAD

**Discussion by diaczykmd** [sumatrapdfreader/sumatrapdf#4405] [Q&A]
I have user try to print a multi-page pdf it only prints one page.  Why does this happen?  This just started this week with multiple users.  These users are using windows 10.   How do I trouble shoot this.   They have other reports that are printing correcting.

**Comment by GitHubRulesOK** [comment]
Perhaps a Windows 10 update ?
I am using 10 with all updates and having a different problem its not even printing one page. 

HOWEVER same identical version freshly installed on Windows 7 same file same printer is working perfectly

![Untitled1](https://github.com/user-attachments/assets/576bb6b1-2aad-4692-a24c-f3076cecb9a0)
currently ejecting page 5

Ahh the windows 10 printer driver was not set to an active port so that is now working. 
Probably some past messing about on my part using test files!

**Comment by kjk** [comment]
There are many ways to print. I need detailed, step-by-step explanation of all the steps to reproduce this, preferably with screenshots.
